src/org/thoughtcrime/securesms/glide/ContactPhotoFetcher.java

```diff
@@ -31,7 +31,7 @@ public void loadData(Priority priority, DataCallback<? super InputStream> callba
       inputStream = contactPhoto.openInputStream(context);
       callback.onDataReady(inputStream);
     } catch (IOException e) {
-      callback.onLoadFailed(e);
+      //callback.onLoadFailed(e);//TODO dbg
```

Comment:
What is this todo?

Reply:
Ah yes, I had disabled this line because I suspected it to be noisy. But I'll revert it, maybe I'll redo this in another PR but this needs some thoughts about side effects first.